packages/backend/server/src/app.ts

```diff
@@ -22,6 +23,8 @@ export async function createApp() {
     bodyParser: true,
```

Comment:
Add `bufferLogs: true` to `NestFactory.create` options should be better, follow the document example https://docs.nestjs.com/techniques/logger#dependency-injection